frame/vamm/src/tests/mod.rs

```diff
@@ -79,21 +50,70 @@ struct TestSwapConfig<VammId, Balance> {
 	output_amount_limit: Option<Balance>,
 }
 
-#[allow(dead_code)]
 const ZERO_RESERVE: Balance = Balance::MIN;
-
-#[allow(dead_code)]
 const MINIMUM_RESERVE: Balance = ZERO_RESERVE + 1;
-
-#[allow(dead_code)]
 const MAXIMUM_RESERVE: Balance = Balance::MAX;
-
-#[allow(dead_code)]
 const RUN_CASES: u32 = 1000;
 
+// ----------------------------------------------------------------------------------------------------
```

Comment:
if you find the amount of helper functions keeps growing, I would suggest making a helper.rs file.

Reply:
5 is enough to justify keeping it inside of mod.rs, thoguh

Reply:
I might implement that idea too